src/cert.rs

```diff
@@ -46,6 +46,23 @@ impl Cert {
     }
 }
 
+#[derive(Clone, Default)]
```

Comment:
I think this can also be `Copy` since it's just a small enum:

```suggestion
#[derive(Clone, Copy, Default)]
```